### ISSUE TITLE
What is the best server to server communication method?

### ISSUE THREAD

**Issue by mandolout** [labrad/pylabrad#269]
**what is the appropriate way for a labrad server to send a command (setting) to another labrad server?**

I am making a labrad server that lets the user give names to the settings of device servers, then access those settings(set values / get values) by name. In order to do this, the server needs to be able to communicate with the other running servers; specifically it needs to be able to access other servers' settings.
The way I have been achieving this is through the server's connection to labrad: self.client (EG, self.client.device_server.setting(...)); I was wondering if this is the appropriate method to use or if there is a better way for servers to communicate with each other.

Thank you

**Comment by maffoo** [comment]
Yes, using the `client` attribute inside a server is the easiest way to do this. You can see lots of examples of this if you poke around our servers repository: [martinisgroup/servers](https://github.com/martinisgroup/servers)
